src/components/Integrations/Table.tsx

```diff
@@ -382,14 +364,12 @@ export const IntegrationsTable: React.FunctionComponent<
         {...getOuiaProps('Integrations/Table', { ...props, ouiaSafe: false })}
       >
         <SkeletonTable
```

Comment:
We could use implementation from the component-groups, but it can be done as a follow-up